docs/source/verdi/verdi_user_guide.rst

```diff
@@ -652,8 +652,8 @@ Below is a list with all available subcommands.
                                       superuser.
 
       --repository DIRECTORY          Absolute path to the file repository.
-      --config FILE                   Load option values from configuration file
-                                      in yaml format.
+      --config URLFILE                Load option values from configuration file
```

Comment:
One question I'd like to know, this metavar `URLFILE` is gotten set in the class scope variable `name` of the click type class? And will be convert to upper case in cmdline interface automatically?

Reply:
The answer to your questions are: yes and yes.

Now that you mention it, I think `FILEURL` would be a better name for the metavar and the class (I originally started from modifying `ImportPath` => `UrlPath` => `UrlFile`).
Also URL is an acronym in itself, so it should be captitalized in the class name.
Fixed!